### PR TITLE
docs: AKS-engine (on Stack) considerations around the use of proxy servers

### PR DESCRIPTION
**Pull request by heoelri**
**Reason for Change**:

This PR contains a few considerations and guidance about the use of proxy servers in connected and disconnected scenarios of Azure and Azure Stack Hub. This applies to AKS-engine as well as (at least partially) to AKS and Kubernetes in general.

**Issue Fixed**:

- no issues addressed in this PR

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: docs/topics/README.md [hunk 1]
```diff
@@ -11,15 +11,25 @@ Introductions to all the key parts of AKS Engine you’ll need to know.
...
 
+**Operations**
+
+- [Scaling Kubernetes Clusters](scale.md)
+- [Upgrading Kubernetes Clusters](upgrade.md)
+- [Monitoring Kubernetes Clusters](monitoring.md)
+
+**Azure Stack**
+
+Next using AKS-engine in Azure there are some specific considerations for Azure Stack:
```
Comment (jadarsie) [inline docs/topics/README.md:28]: I think `AKS Engine` is preferred over `AKS-engine`

Comment (heoelri) [reply docs/topics/README.md]: changed it from aks-engine to aks engine

File: docs/topics/proxy-servers.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
+# AKS Engine and proxy servers
+
+Using AKS engine on [Azure Stack](azure-stack.md) in connected and disconnected environments often times requires the use of a non-transparent proxy server. Non-transparent means that they're not part of the default route and have to be configured to be used.
```
Comment (jackfrancis) [inline docs/topics/proxy-servers.md:3]: nit: use "AKS Engine" everywhere (instead of lowercase "AKS engine")

Comment (heoelri) [reply docs/topics/proxy-servers.md]: done, replace AKS engine with AKS Engine (with an uppercase E)

**Pull request merged**